_posts/2016-12-19-devlab-v3-is-here.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Devlab v3 Is Here"
+date: 2015-12-19 8:00:00
+categories: docker devlab development
+author: kent.safranski
+cover: /assets/images/covers/container-load.jpg
+description: After a year of dogfooding, a complete rebuild, and 110+ commits, we are excited to release Devlab v3.
+comments: true
+---
+
+After a year of dogfooding, a complete rebuild, and 110+ commits, we are excited to release Devlab v3.
+
+## New to Devlab?
+
+Devlab is a CLI tool that allows you to easily spin up [Docker](https://www.docker.com/) containers with all the services, environment variables, and settings you need to run a project in a "cleanroom".
+
+Simply put, if you want to run an application you're developing that requires a database (or any other services), you simply define the services, the configuration, and tasks you routinely run. Once your project is configured you run `tasks` and every time Devlab spins up a parent container and links in any service containers you need.
```

Comment:
A couple things in the first sentence:
- I'd probably swap out the second `simply`, since the word's already being used at the beginning
- And in `you [simply] define the services`, it should probably be `service` since (despite the parenthetical) you're only referring to a single database

And in the second sentence:
- I'd either throw a comma in after `every time`, because without it the flow of the sentence sort of makes it seem like an incomplete thought (even though grammatically it does make sense)